### ISSUE TITLE
On mobile update icon

### ISSUE THREAD

**Issue by nitanagdeote** [datavis-tech/vizhub-issue-tracker#523]
In editor mode,update icon is on the left side under the viz window,usually it is on right.

**Comment by curran** [comment]
Good catch! I think we can do this one. Agree that consistency would be great.

**Comment by curran** [comment]
To clarify visually, I think this image captures the desired design for mobile:

![image](https://user-images.githubusercontent.com/68416/118373246-cb3c4500-b583-11eb-8521-41d29cc233a5.png)
